resources/lang/en/sort.php

```diff
@@ -16,25 +18,15 @@
  *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-.beatmapsets-sorting {
-  font-size: @font-size--normal;
-  margin: 0 -10px 10px;
-
-  &__item {
-    .link-gray-dark();
-    margin: 0 10px;
-
-    &--selected {
-      color: @pink-dark;
-      font-weight: bold;
+return [
+    '_' => 'Sort by',
 
-      .link-hover({
-        color: @pink-dark;
-      });
-    }
-  }
+    'new' => 'Recent',
+    'old' => 'Old',
+    'top' => 'Top',
 
-  &__item-arrow {
-    margin-left: 5px;
-  }
-}
+    'forum_topics' => [
+        'new' => 'Last replied',
```

Comment:
"Last replied" kinda implies _my_ last reply, so maybe "Last reply"? (also might want to update the topic listing to match too)